airflow/models/taskinstance.py

```diff
@@ -571,7 +571,11 @@ def refresh_from_task(self, task, pool_override=None):
         self.run_as_user = task.run_as_user
         self.max_tries = task.retries
         self.executor_config = task.executor_config
-        self.operator = task.__class__.__name__
+        if task.__class__.__name__ != "SerializedBaseOperator":
+            self.operator = task.__class__.__name__
+        else:
+            self.operator = task.task_type
```

Comment:
I think this can simply be 
```suggestion
        self.operator = task.task_type
```

As BaseOperator already has a task_type property